server/routes/visits.ts

```diff
@@ -26,44 +26,71 @@ export default function routes({ auditService, visitService, visitSessionsServic
 
     const { type = '', sessionReference = '', selectedDate = '', firstTabDate = '' } = req.query
 
-    const selectedType: VisitRestriction = type === 'OPEN' || type === 'CLOSED' || type === 'UNKNOWN' ? type : 'OPEN'
+    const selectedSessionReference = sessionReference.toString()
+    const selectedType: VisitRestriction = type === 'OPEN' || type === 'CLOSED' || type === 'UNKNOWN' ? type : undefined
```

Comment:
Wonder if it might read better as `['OPEN', 'CLOSED', 'UNKNOWN'].includes(type)` Personal preference though.

Reply:
Yeah, it's not particularly pretty. I just tried your suggestion which initially looked nicer but it turns out it doesn't actually narrow the type for Typescript 🙁 . It doesn't seem to realise that `type` must now be one of the three values, so complains of type mismatch between `selectedType` and `type`!

Reply:
Go TypeScript 😆